src/index.html

```diff
@@ -1,11 +1,25 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>Document Two Sections</title>
     <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Excepteur Sint Occaecat</h1>
+    <!-- This is the sidebar area-->
```

Comment:
No need for comments like this because it's obvious from the class name it is a sidebar

Usually we use comments to provide context and information that isn't quite apparent when you read the code.

Things like gotchas or I had to do this way instead of the better way due to this particular constraint, bla, bla, bla.

Reply:
Ok thank you I understand much more now the use of comments